src/stylesheets/darwin.scss

```diff
@@ -49,13 +49,18 @@ $mode: "production" !default;
 
 .math-node.empty-math .math-render::before {
 	content: "Click to insert LaTex ...";
+    color: black;
 }
 
 .math-node .math-render.parse-error::before {
 	content: "Cannot parse LaTeX.";
 	cursor: help;
 }
 
+.math-node .ProseMirror-focused {
+    outline: none;
+}
+
 math-display {
     position: relative;
 
```

Comment:
I guess we can just move all the math related SCSS into `components/core/math.scss`. From both `darwin.scss` and `generic.scss`.